README.md

```diff
@@ -1,29 +1,75 @@
-jmx4prometheus
+JMX Exporter
 =====
 
-JMX to prometheus bridge.
+JMX to Prometheus bridge.
 
-A webserver that exposes mBeans of a JMX target in JSON format suitable
-for http://github.com/prometheus.
-
-# About
-A prometheus server makes a JSON request to a the jmx4prometheus server which then
-scrapes all beans of a target JMX port on another process. The target can
-either be statically configured in the configuration or passed in by
-prometheus server as a query string parameter.
-Jmx4prometheus can return all mBeans (that are numeric) or be configured with
-whitelist/blacklist behaviour.
+A Collector that can configurably scrape and expose mBeans of a JMX target.
+This can be run as an independant HTTP server, and scrape remote JMX targets.
```

Comment:
"independent"

"HTTP server," -> actually no comma after this one for a change ;) (it's neither a list of three or more items, nor is the thing after the comma a whole sentence by itself)